lib/sequel/model/associations.rb

```diff
@@ -3552,11 +3552,11 @@ def eager_graph_loader
         end
 
         # Eagerly load all specified associations.
-        def eager_load(a, eager_assoc=@opts[:eager])
+        def eager_load(a, m=model, eager_assoc=@opts[:eager])
```

Comment:
When adding an optional argument to an existing method, the optional argument should always be added at the end.